test/integration/client/client_test.go

```diff
@@ -1342,3 +1347,59 @@ func TestExtractModifyApply_ForceOwnership(t *testing.T) {
 		t.Errorf("createMgrExtracted apply configuration did not match expected, got:\n%s\n", cmp.Diff(expectedCreateExtracted, createMgrExtracted))
 	}
 }
+
+func TestAPIServerTransportMetrics(t *testing.T) {
+	result := kubeapiservertesting.StartTestServerOrDie(t, nil, nil, framework.SharedEtcd())
+	defer result.TearDownFn()
+
+	c := clientset.NewForConfigOrDie(result.ClientConfig).RESTClient()
+
+	body, err := c.Get().AbsPath("/metrics").DoRaw(context.Background())
```

Comment:
Before checking the metrics it would probably be good to create one of every resource via `GetEtcdStorageData` to exercise all the common code paths.

Reply:
copied pasted from other test, not influence at all, that is good